presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -180,7 +180,25 @@ protected Boolean visitAtTimeZone(AtTimeZone node, Void context)
         @Override
```

Comment:
Pardon, which one? From the context GH gave me I don't know, but I bet on "Analyze SELECT's Subquery" :)